.gitlab-ci.yml

```diff
@@ -5,16 +5,16 @@ stages:
   - trigger
 
 include:
-  - project: 'Northern.tech/Mender/mendertesting'
-    file: '.gitlab-ci-check-golang-lint.yml'
-  - project: 'Northern.tech/Mender/mendertesting'
-    file: '.gitlab-ci-github-status-updates.yml'
-  - project: 'Northern.tech/Mender/mendertesting'
-    file: '.gitlab-ci-check-commits.yml'
-  - project: 'Northern.tech/Mender/mendertesting'
-    file: '.gitlab-ci-check-license.yml'
+  - project: "Northern.tech/Mender/mendertesting"
+    file: ".gitlab-ci-check-golang-lint.yml"
+  - project: "Northern.tech/Mender/mendertesting"
+    file: ".gitlab-ci-github-status-updates.yml"
+  - project: "Northern.tech/Mender/mendertesting"
+    file: ".gitlab-ci-check-commits.yml"
+  - project: "Northern.tech/Mender/mendertesting"
+    file: ".gitlab-ci-check-license.yml"
 
-image: golang:1.14-alpine3.11
+image: golang:1.18-alpine3.16
```

Comment:
As discussed in person, please move all non-feature related changes to separate commits. At least this one could go on its own (chore: update golang version) and the yaml formatting could also go on their own.

Reply:
done!